tests/resources/test_data_source_dir/test_postgres_data.py

```diff
@@ -0,0 +1,348 @@
+#
+# Copyright (c) 2012-2025 Snowflake Computing Inc. All rights reserved.
+#
+
+import datetime
+from decimal import Decimal
+
+from snowflake.snowpark.types import (
+    StructType,
+    StructField,
+    StringType,
+    DecimalType,
+    DoubleType,
+    LongType,
+    BinaryType,
+    BooleanType,
+    VariantType,
+    DateType,
+    TimestampType,
+    TimeType,
+    TimestampTimeZone,
+)
+
+POSTGRES_TABLE_NAME = "test_schema.ALL_TYPE_TABLE"
+EXPECTED_TEST_DATA = [
+    (
+        -6645531000000000000,
```

Comment:
we should update the test data after mysql PR that have base_driver change merged.
this data could not test whether precision is lost. For example, if the data is 6645531000000000111, you can tell if the precision is preserved by the last three digits

Reply:
test updated to change last 3 digits